windows/src/engine/keyman32/tests/keyman-engine-tests/appinttests.cpp

```diff
@@ -78,3 +78,58 @@ TEST(AppContext, AppContext_Delete) {
   testContext.Delete();
 
 }
+
+// Test normalize and restore line break functions
```

Comment:
I think we need some tests for surrogate pair edge cases, especially boundary conditions